components/common/layout/PageLayout.tsx

```diff
@@ -32,14 +32,15 @@ const PageLayout: FC<PageLayoutProps> = ({
 }): ReactElement => {
   return (
     <div>
-        {!blogPage
-          && <OgData
-            ogTitle={seoData.title || 'Open Sauced'}
-            ogDescription={seoData.description || ''}
-            ogImageUrl={seoData.image as unknown as string || ''}
-            ogUrl={seoData.url || 'https://opensauced.pizza'}
-            />
-        }
+      {!blogPage
+        && <OgData
+        ogTitle={seoData.title || 'Open Sauced'}
+        ogDescription={seoData.description || ''}
+        ogImageUrl={seoData.image as unknown as string || ''}
+        ogUrl={seoData.url || 'https://opensauced.pizza'}
+        noindex={false} 
+        linkHref={''} />
```

Comment:
You don't need to pass this since it's an empty string

Reply:
> You don't need to pass this since it's an empty string

Tried something like this it'll trow an eslint error